platform/linux-generic/odp_schedule_basic.c

```diff
@@ -2253,3 +2367,42 @@ const _odp_schedule_api_fn_t _odp_schedule_basic_api = {
 	.schedule_order_wait      = order_lock,
 	.schedule_print           = schedule_print
 };
+
+/* API functions used when powersave is enabled in the config file. */
+const _odp_schedule_api_fn_t _odp_schedule_basic_sleep_api = {
+	.schedule_wait_time       = schedule_wait_time,
+	.schedule_capability      = schedule_capability,
+	.schedule_config_init     = schedule_config_init,
+	.schedule_config          = schedule_config,
+	/*
+	 * The functions here are the same as above, except for the *_sleep
+	 * functions below.
+	 */
```

Comment:
Something like this could be a bit clearer:
```
	/* Only the following *_sleep functions differ from _odp_schedule_basic_api */
	.schedule                 = schedule_sleep,
	.schedule_multi           = schedule_multi_sleep,
	.schedule_multi_wait      = schedule_multi_wait_sleep,
	/* End of powersave specific functions */
```

Reply:
v4.